pkg/sql2pgroll/alter_table.go

```diff
@@ -442,6 +565,24 @@ func getQualifiedRelationName(rel *pgq.RangeVar) string {
 	return fmt.Sprintf("%s.%s", rel.GetSchemaname(), rel.GetRelname())
 }
 
+// canConvertColumnDef returns true iff `col` can be converted to a pgroll `Column` definition.
+// TODO: Copied from WIP PR
+func canConvertColumnDef(col *pgq.ColumnDef) bool {
```

Comment:
Another copy of this function was merged in #548. We should rebase this PR and use it.